libs/data-display/src/useSortableDataTable.tsx

```diff
@@ -0,0 +1,30 @@
+import { useDataTable } from "./index";
+
+export default function useSortableDataTable<T, U extends keyof T>(initialData: T[], columnMapping: Record<U, string>) {
+  const [dataTableState, dataTableHook] = useDataTable();
+
+  const generateSortedData = () => {
```

Comment:
I think this should be a useMemo instead of just function. Also, use useMemo for the return object because you're recreating it on every rerender